apps/web/public/static/locales/en/common.json

```diff
@@ -2465,7 +2465,6 @@
   "lowest_rated_members": "Members with lowest rated meetings",
   "csat_score": "CSAT Score",
   "lockedSMS": "Locked SMS",
-  "signing_up_terms": "By proceeding, you agree to our <1>Terms</1> and <2>Privacy Policy</2>.",
```

Comment:
I thought I left a comment somewhere. Anyway here it goes again, `signing_up_terms` has different values, could you confirm to me you have deleted the correct one.

A screenshot would do. Also, please make sure you restart dev server after the common.json is updated.

Reply:
Sure @hariombalhara , so we deleted correct one now. We get terms as first element in index so keeping message with <0>Terms</0> is correct and removing key with message <1>Terms</1>.

Sorry but screenshot of what ?

Reply:
Sure @hariombalhara , so we deleted correct one now. We get terms as first element in array so keeping message with <0>Terms</0> is correct and removing key with message <1>Terms</1>.

Here's video: https://www.loom.com/share/eb3d5f600bcb492e92d31c6b94a9853e
If we keep message with <1>Terms</1> Privacy Policy will not be valid. Only terms will get highlighted.

Reply:
Thanks, great explanation as well !!

Reply:
Welcome. Thank you.

Reply:
Welcome, Thank you.